app/views/layouts/printer_friendly.html.haml

```diff
@@ -1,11 +1,13 @@
 !!!
 %html{dir:language_direction}
-  %head
+  %head      
     %title
       =h html_title
     %style
       a.anchor, a.shy {display:none;}
+      @media print { .noPrint {display: none}}
   %body{class:language_direction}
+    %button{:onclick => 'window.history.back()', :class => 'noPrint'} back
```

Comment:
you could also use
```haml
%button.no-print{:onclick => 'window.history.back()'} back
```
And we could use a class and add the onclick hook on dom loaded instead. But that's for another time i guess.